grapple/types/redirects.py

```diff
@@ -35,11 +36,26 @@ def resolve_old_url(self, info, **kwargs) -> str:
 
     def resolve_new_url(self, info, **kwargs) -> Optional[str]:
```

Comment:
suggestion: it may be nice to type hint this a tad - I have to admit it took me a bit to go "ah, `self` in this context is a `Redirect` instance" 🤦🏼 

```suggestion
    def resolve_new_url(self: Redirect, info, **kwargs) -> Optional[str]:
```

Reply:
Thanks Dan, that's a really good point, it usually gets me too. Added in e2f0a419e9a0e5b603965a92bea249dd20b7ebbf